[2022-06-12 16:43:30,840] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-12T01:35:00+00:00 [queued]>
[2022-06-12 16:43:30,901] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-12T01:35:00+00:00 [queued]>
[2022-06-12 16:43:30,901] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-12 16:43:30,902] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-12 16:43:30,902] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-12 16:43:30,961] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): save_data_task> on 2022-06-12 01:35:00+00:00
[2022-06-12 16:43:30,967] {standard_task_runner.py:52} INFO - Started process 5171 to run task
[2022-06-12 16:43:30,975] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'ingest_data', 'save_data_task', 'scheduled__2022-06-12T01:35:00+00:00', '--job-id', '585', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_dag.py', '--cfg-path', '/tmp/tmplzk089s8', '--error-file', '/tmp/tmpemrgvlkg']
[2022-06-12 16:43:30,977] {standard_task_runner.py:80} INFO - Job 585: Subtask save_data_task
[2022-06-12 16:43:31,114] {task_command.py:370} INFO - Running <TaskInstance: ingest_data.save_data_task scheduled__2022-06-12T01:35:00+00:00 [running]> on host DESKTOP-GDPK8BL.localdomain
[2022-06-12 16:43:31,579] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=save_data_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-12T01:35:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-12T01:35:00+00:00
[2022-06-12 16:43:31,593] {python.py:173} INFO - Done. Returned value was: None
[2022-06-12 16:43:31,625] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=save_data_task, execution_date=20220612T013500, start_date=20220612T144330, end_date=20220612T144331
[2022-06-12 16:43:31,714] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-12 16:43:31,908] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
